src/main/java/org/broadinstitute/hellbender/tools/walkers/SplitIntervals.java

```diff
@@ -86,11 +86,14 @@
 
     public static final String MIN_CONTIG_SIZE_LONG_NAME = "min-contig-size";
 
+    public static final String INTERVAL_FILE_PREFIX_FULL_NAME = "interval-file-prefix";
     public static final String INTERVAL_FILE_EXTENSION_FULL_NAME = "extension";
 
+    public static final String DEFAULT_PREFIX = "";
     public static final String PICARD_INTERVAL_FILE_EXTENSION = "interval_list";
     public static final String DEFAULT_EXTENSION = "-scattered." + PICARD_INTERVAL_FILE_EXTENSION;
 
+    public static final String INTERVAL_NUMBER_OF_DIGITS_FULL_NAME = "interval-file-num-digits";
```

Comment:
this is the only flag currently being yelled about, so maybe we'll limit it to just this for now